### ISSUE TITLE
UrlFormat: Fails on replace pattern

### ISSUE THREAD

**Issue by chris-dura** [brigade/scss-lint#699]
So, I'm in the process of moving away from Compass, and one of the things Compass did for me was insert some cache busters into my CSS resources like fonts and images.
I know there's several alternative ways of doing this, but the one that seemed the most simple was to just put a pattern into the url in the scss file that can easily be replaced via a gulp task...
```scss
.hello {
    background-image: url('../images/world.png?{{CACHE-VERSION}}');
}
```
```js
gulp.task('styles', function() {
        return gulp
            .src(./sass/**/*.scss)
            .pipe(sass(sassOptions).on('error', sass.logError))
            .pipe(replace('{{CACHE-VERSION}}', 'v=' + packageJson.version))
            .pipe(autoprefixer(autoprefixerOptions))
            .pipe(gulp.dest(output));
    });
```

However, I'm getting a "bad URI(is not URI?)" error from `UrlFormat`. The documentation says the `UrlFormat` is just to make sure you don't have extra info like protocol and domain in URLs, but it's obviously checking for valid URI formatting as well.

Should `scss_lint` allow this kind of thing, since I'm not technically violating the linter?
Or, is there a different pattern that I could use (maybe without `{{}}`) that would be easily replaced, but not cause an invalid URI warning?

**Comment by sds** [comment]
Hey @chris-dura,

Curly braces [must be escaped in valid URIs](http://stackoverflow.com/a/6742274), so you _are_ technically violating the linter in this case.

Is there anything wrong with just replacing the string `CACHE-VERSION` (without braces)?

**Comment by chris-dura** [comment]
>  so you are technically violating the linter in this case.
Yeah, I just meant the documentation didn't indicate that the linter was checking for validity of the URL, just that protocols and domains should not be in there.

> Is there anything wrong with just replacing the string CACHE-VERSION (without braces)?
probably not.

Thanks!
